### PR TITLE
feat: Implemented getCode.

### LINKED ISSUE #2276: Implement `getCode(address, blockTag)` :in the websocket server
### Problem

When using websockets some JSON RPC methods are not available. `getCode(address, blockTag)` is one of them.

### Solution

Enhance the outer switch, case, statement in the webSocketServer.ts to call the `getCode(address, blockTag)` method in the Relay.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ebadiere**
Implemented `eth_getCode` over websockets.

**Related issue(s)**:

Fixes #2276

### CODE REVIEW

File: packages/ws-server/src/webSocketServer.ts [hunk 1]
```diff
@@ -169,6 +169,16 @@ app.ws.use(async (ctx) => {
             connectionIdPrefix,
           );
           break;
+        case WS_CONSTANTS.METHODS.ETH_GET_CODE:
+          relayResponse = await relay.eth().getCode(params[0], params[1]);
```
Comment (quiet-node) [inline packages/ws-server/src/webSocketServer.ts:173]: Why didn't you use the `response` variable init at line 131? Using the same `response`you wouldn't have to send the result back to the client again yourself but down below (after the switch block) it already send back to the client. 

```suggestion
const contractCode = await relay.eth().getCode(params[0], params[1]);
response = jsonResp(request.id, null, contractCode)
```

**Review by quiet-node — CHANGES_REQUESTED** [review]

File: packages/ws-server/src/webSocketServer.ts (hunk 1 above)
Comment (ebadiere) [reply packages/ws-server/src/webSocketServer.ts:173]: If I do that it falls into the if statement on line 194, which will not format the response correctly, or at least match the ethereum RPC's response:
```
    if (response) {
      ctx.websocket.send(JSON.stringify(response));
    }
```

**Review by quiet-node — APPROVED** [review]

**Review by quiet-node — CHANGES_REQUESTED** [review]

File: packages/ws-server/src/webSocketServer.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/ws-server/src/webSocketServer.ts]: Fixed.

File: packages/ws-server/src/controllers/eth_getCode.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ *
+ */
+
+import { sendToClient } from '../utils/utils';
+import { Relay } from '@hashgraph/json-rpc-relay';
+import { validateParamsLength } from '../utils/validators';
+import { handleSendingTransactionRequests } from './helpers';
+
+/**
+ * Handles the "eth_getTransactionByHash" method request by retrieving transaction details from the Hedera network.
```
Comment (quiet-node) [inline packages/ws-server/src/controllers/eth_getCode.ts:27]: You might want to update the doc comment

**Review by quiet-node — APPROVED** [review]
LG one nit

File: packages/ws-server/src/controllers/eth_getCode.ts (hunk 2 above)
Comment (ebadiere) [reply packages/ws-server/src/controllers/eth_getCode.ts]: Done.

**Review by quiet-node — APPROVED** [review]
LGTM

**Pull request merged**